server/src/main/java/org/elasticsearch/action/support/master/TransportMasterNodeAction.java

```diff
@@ -101,8 +103,19 @@ protected TransportMasterNodeAction(Settings settings, String actionName, boolea
 
     protected abstract String executor();
 
+    /**
+     * Returns a new response instance for serialization. This is deprecated; new implementors
```

Comment:
Usually you'd add the `@deprecated` javadoc tag instead of saying "this is deprected". Yes, I know there are not two tags, but that is how the cookie crumbles.